File: 2° Periodo/Prog 2  JAVA C++/POO_TRABALHO_1/src/Medico.java
public class Medico extends Pessoa {
    private int crm;
    private String especialidade;
    

    public Medico() {
    }

    public Medico(String nome, String endereco, String cpf, char sexo, int telefone, int identidade, int crm, String especialidade) {
        super(nome, endereco, cpf, sexo, telefone, identidade);
        this.crm = crm;
        this.especialidade = especialidade;
    }

    public void imprimir(){
        super.imprimir();
        System.out.println("CRM: "+crm);
        System.out.println("Especialidade: "+ especialidade);
    }

    public int getCrm() {
        return crm;
    }

    public void setCrm(int crm) {
        this.crm = crm;
    }

    public String getEspecialidade() {
        return especialidade;
    }

    public void setEspecialidade(String especialidade) {
        this.especialidade = especialidade;
    }

}

File: 2° Periodo/Prog 2  JAVA C++/LAB 10/src/Medico.cpp
#include "Medico.hpp"

Medico::Medico() {}

Medico::Medico(string nome, char sexo, string endereco, int cpf, string telefone, string identidade, int CRM, string especialidade)
    : Pessoa(nome,sexo,endereco,cpf,telefone, identidade) {
        this->CRM = CRM;
        this->especialidade = especialidade;
}

void Medico::imprimir() {
    Pessoa::imprimir();
    cout << "CRM: " << CRM << endl;
    cout << "Especialidade: " << especialidade << endl;
}

int Medico::getCRM() {
    return CRM;
}

void Medico::setCRM(int crm) {
    CRM = crm;
}

string Medico::getEspecialidade() {
    return especialidade;
}

void Medico::setEspecialidade(string especialidade) {
    this->especialidade = especialidade;
}